### ISSUE TITLE
Not allowed to modify image with different name

### ISSUE THREAD

**Issue by veger** [refinery/refinerycms#3279]
After fixing #3278 I noticed that uploading a new image with a different name is not allowed. After a quick scan I found that this is indeed refused since 2287d76e0096401458ca0022b5e5f276bf5c496b.

I have disabled the validation rule as a test to see what would go wrong, but everything works as expected: old image is removed (in image store) , new image is added, refinery/admin shows new image, etc.

My question now is: why is this disabled/blocked??? Was there a problem (commit is quite old), which is fixed now?

**Comment by bricesanchez** [comment]
I have absolutly no idea but i'm ok to remove this validation if @parndt is ok too?
Did i miss something?

**Comment by parndt** [comment]
It's disabled because references to the image that are inserted into WYSIWYG components become 404s

**Comment by bricesanchez** [comment]
>It's disabled because references to the image that are inserted into WYSIWYG components become 404s

You're right, i missed this!

So, we could : 
1. Don't accept this feature for now due to this restriction
2. Add a feature to give the ability to update the path of assets in textarea WYSIWYG fields, like this guide but with a worker: https://www.brownwebdesign.com/blog/refinery-update-assets
3. Refactor the way we embed pictures in a textarea with the WYSIWYG in order to reference an id instead an hardcoded source: `<img id=1>` will be build to `<img src="/system/.../1.jpg" alt="alt 1" title="title 1" />` with JS in backend and with a Presenter in frontend.

I don't like the point 2. because it could mess a website if we made a mistake. 
Point 3., could be cool but we need someone to create this feature.
